Assets/HoloToolkit-Preview/QRTracker/Scripts/QRCodesManager.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections;
+
+using System.Collections.Generic;
+
+using UnityEngine;
+
+using QRCodesTrackerPlugin;
+namespace HoloToolkit.Unity.QRTracking
+{
+    public static class QRCodeEventArgs
+    {
+        public static QRCodeEventArgs<TData> Create<TData>(TData data)
+        {
+            return new QRCodeEventArgs<TData>(data);
+        }
+    }
+
+    [Serializable]
+    public class QRCodeEventArgs<TData> : EventArgs
```

Comment:
classes should be in their own files

Reply:
not 100% required for a preview feature in the HTK